README.md

```diff
@@ -38,25 +38,19 @@
 </div>
 <br/>
 
-## Usage
+Iroh is a protocol for syncing & moving bytes. Bytes of any size, on any device. At it's core, it's a peer-2-peer network built on a _magic socket_ that establishes [QUIC](https://en.wikipedia.org/wiki/QUIC) connections between peers. Peers request and provide _blobs_ of opaque bytes that are incrementally verified by their BLAKE3 hash during transfer.
+
+Iroh _peers_ create _documents_: mutable maps that connect arbitrary keys to BLAKE3 hashes of blobs. Documents can have multiple _editors_ and any number of _readers_. Online peers subscribe to changes that document editors publish for live, incremental updates. Peers sync documents quickly, and fetch linked blobs lazily.
```

Comment:
This code is not merged yet, lets keep documents and editors out of this release please